packages/foam-vscode/static/dataviz/graph.js

```diff
@@ -205,7 +205,7 @@ function initDataviz(channel) {
     )
     .nodeCanvasObject((node, ctx, globalScale) => {
       const info = model.graph.nodeInfo[node.id];
-      if (info == null) {
+      if (info === null) {
```

Comment:
`== null` is different from `=== null`.  The former is also `true` against `undefined`

`undefined === null  => false`
`undefined == null => true`

That is why for these checks we use `==`

Reply:
I'm aware of the difference, I've just assumed that it was a stylistical matter here because the same style is used all over the place, in most cases it doesn't seem to have any semantic motivation, but if there is any doubt regarding that I prefer to revert all those changes altogether.

Reply:
yeah, if there is no `model.graph.nodeInfo[node.id]`, `info` would be `undefined`.

but the places where you changed it for the string comparison I think look good, I would keep those